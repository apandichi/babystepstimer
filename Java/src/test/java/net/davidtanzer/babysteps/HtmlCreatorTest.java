package net.davidtanzer.babysteps;

import org.approvaltests.Approvals;
import org.junit.Test;

public class HtmlCreatorTest {
    private HtmlCreator htmlCreator = new HtmlCreatorImpl();

    @Test
    public void testTimerHtml() {
        String first = htmlCreator.createTimerHtml("first random text", "first body color", true);
        String second = htmlCreator.createTimerHtml("second random text", "second body color", false);
        Approvals.verify(first + "\n" + second);
    }
}
