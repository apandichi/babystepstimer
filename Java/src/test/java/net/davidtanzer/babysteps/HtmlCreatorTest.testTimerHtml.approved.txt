<html><body style="border: 3px solid #555555; background: first body color; margin: 0; padding: 0;"><h1 style="text-align: center; font-size: 30px; color: #333333;">first random text</h1><div style="text-align: center"><a style="color: #555555;" href="command://stop">Stop</a> <a style="color: #555555;" href="command://reset">Reset</a> <a style="color: #555555;" href="command://quit">Quit</a> </div></body></html>
<html><body style="border: 3px solid #555555; background: second body color; margin: 0; padding: 0;"><h1 style="text-align: center; font-size: 30px; color: #333333;">second random text</h1><div style="text-align: center"><a style="color: #555555;" href="command://start">Start</a> <a style="color: #555555;" href="command://quit">Quit</a> </div></body></html>